include/grpc/impl/codegen/grpc_types.h

```diff
@@ -261,6 +261,10 @@ typedef enum grpc_call_error {
   GRPC_CALL_ERROR_PAYLOAD_TYPE_MISMATCH
 } grpc_call_error;
 
+/* Default send/receive message size limits in bytes. -1 for unlimited. */
+#define GRPC_DEFAULT_MAX_SEND_MESSAGE_LENGTH -1
```

Comment:
Please add a TODO(roth) indicating that this value will be changed to match the default receive limit after the next gRPC release.